repose-aggregator/components/filters/header-translation-filter/src/main/resources/META-INF/schema/config/header-translation.xsd

```diff
@@ -39,7 +39,7 @@
         <xs:assert vc:minVersion="1.1"
                    test="count(distinct-values(tokenize(lower-case(string-join(
                    (header-translation:header/@original-name),',')),',')))
```

Comment:
Why do we join then split the names? Does the following not work:
`count(distinct-values(header-translation:header/@original-name)) = count(header-translation:header/@original-name)`